sources/packages/backend/apps/api/src/route-controllers/models/pagination.dto.ts

```diff
@@ -24,9 +24,10 @@ abstract class PaginationOptionsAPIInDTO {
   page: number;
   /**
    * Page size or records per page.
+   * -1 means all records.
    */
-  @Min(1)
-  @Max(50)
+  @Min(-1)
+  @Max(100)
```

Comment:
Why did we change this to 100?

Reply:
because the default pagination has `100`